apps/web/src/app/billing/organizations/billing-sync-api-key.component.html

```diff
@@ -1,117 +1,94 @@
-          <p>{{ "rotateBillingSyncTokenTitle" | i18n }}</p>
-          <app-callout type="warning">
-            {{ "rotateBillingSyncTokenWarning" | i18n }}
-          </app-callout>
-        </ng-container>
+<form [formGroup]="formGroup" [bitSubmit]="submit">
+  <bit-dialog>
+    <h1 bitDialogTitle>
+      {{ (hasBillingToken ? "viewBillingSyncToken" : "generateBillingSyncToken") | i18n }}
+    </h1>
+    <div bitDialogContent>
+      <app-user-verification formControlName="verification" *ngIf="!clientSecret">
+      </app-user-verification>
 
-        <div *ngIf="clientSecret && !showRotateScreen">
-          <p>{{ "copyPasteBillingSync" | i18n }}</p>
-          <label for="clientSecret">{{ "billingSyncKey" | i18n }}</label>
-          <div class="input-group">
-            <input
-              id="clientSecret"
-              class="form-control text-monospace"
-              type="text"
-              [(ngModel)]="clientSecret"
-              name="clientSecret"
-              disabled
-            />
-            <div class="input-group-append">
-              <button
-                type="button"
-                class="btn btn-outline-secondary"
-                (click)="copy()"
-                [appA11yTitle]="'copy' | i18n"
-              >
-                <i class="bwi bwi-lg bwi-clone" aria-hidden="true"></i>
-              </button>
-            </div>
-          </div>
-          <div class="small text-muted mt-2" *ngIf="showLastSyncText">
-            <b class="font-weight-semibold">{{ "lastSync" | i18n }}:</b>
-            {{ lastSyncDate | date : "medium" }}
-          </div>
-          <div class="small text-danger mt-2" *ngIf="showAwaitingSyncText">
-            <i class="bwi bwi-error"></i>
-            {{
-              (daysBetween === 1 ? "awaitingSyncSingular" : "awaitingSyncPlural")
-                | i18n : daysBetween
-            }}
+      <ng-container *ngIf="clientSecret && showRotateScreen">
+        <p>{{ "rotateBillingSyncTokenTitle" | i18n }}</p>
+        <bit-callout type="warning">
+          {{ "rotateBillingSyncTokenWarning" | i18n }}
+        </bit-callout>
+      </ng-container>
+
+      <div *ngIf="clientSecret && !showRotateScreen">
+        <p>{{ "copyPasteBillingSync" | i18n }}</p>
+        <label for="clientSecret">{{ "billingSyncKey" | i18n }}</label>
+        <div class="input-group">
+          <input
+            id="clientSecret"
+            class="form-control text-monospace"
+            type="text"
+            [(ngModel)]="clientSecret"
+            [ngModelOptions]="{ standalone: true }"
+            name="clientSecret"
+            disabled
+          />
+          <div class="input-group-append">
+            <button
+              type="button"
+              class="btn btn-outline-secondary"
+              (click)="copy()"
+              [appA11yTitle]="'copy' | i18n"
+            >
+              <i class="bwi bwi-lg bwi-clone" aria-hidden="true"></i>
+            </button>
```

Comment:
Can we change this to use the bit-form-field?

```suggestion
        <bit-form-field>
          <bit-label>{{ "billingSyncKey" | i18n }}</bit-label>
          <input
            bitInput
            id="clientSecret"
            type="text"
            [value]="clientSecret"
            class="tw-font-mono"
            disabled
          />
          <button
            bitIconButton="bwi-clone"
            bitSuffix
            type="button"
            [appCopyClick]="clientSecret"
            [appA11yTitle]="'copyValue' | i18n"
          ></button>
        </bit-form-field>
```